src/applications/claims-status/containers/AskVAPage.jsx

```diff
@@ -109,23 +126,26 @@ class AskVAPage extends React.Component {
                 </li>
                 <li>The date benefits will begin if we approve your claim</li>
               </ul>
-              <div className="usa-alert usa-alert-info background-color-only claims-alert">
-                <Checkbox
-                  className="claims-alert-checkbox"
-                  checked={this.state.submittedDocs}
-                  onValueChange={update => this.setSubmittedDocs(update)}
-                  label="I have submitted all evidence that will support my claim and I’m not going to turn in any more information. I would like VA to make a decision on my claim based on the information already provided."
+              <div>
+                <VaCheckbox
+                  description=""
```

Comment:
Just curious - should this be an empty string?

Reply:
Probably don't need it